### ISSUE TITLE
NVDA should have a function to detect if hotkeys of newly installed addon are conflicting with the current hotkeys in the gesture setting

### ISSUE THREAD

**Issue by sine-w** [nvaccess/nvda#11255]
### Is your feature request related to a problem? Please describe.

- I first installed ClipContent designer addon. it has a hotkey to add current selection to clipboard by nvda win c
- afterwards, I installed Golderncursor addon 
- then when I press nvda win c and wanna copy to clipboard I found that the hotkey is overwrited by Golderncursor of setting the hotkey as changing the cursor moving unit.

The above is only one of the examples   
### Describe the solution you'd like

NVDA should be able to determine whether newly installed addon has any hotkey that will conflict with the current hotkeys

for example when installing new addon, nvda will cross check the hotkeys of the addons and the current hotkeys, if particular hotkey is being used, NVDA will display an alert listing out which hotkey is having conflict
 and probably recommending user to change one of the hotkey
 

### Describe alternatives you've considered
if NVDA detected conflict
provide an option for user to choose which function should be used for that hotkey 
as the current one, the hotkey of the lastly installed addon will be the active one

**Comment by XLTechie** [comment]
This has been needful for a while. It especially effects users who aren't used 
to having multiple add-ons, and the kind of keyboard conflicts that can result.

I'm not sure how best it should be implemented, but as a start, a dialog that 
warns the user of a conflict and lists the steps necessary to resolve it, would 
be helpful.

Later on, we could morph this into a mechanism for allowing on-the-fly 
re-mapping of the conflicted keys.

Luke

**Comment by josephsl** [comment]
HI, I believe this is a duplicate of an older issue. Thanks.

**Comment by XLTechie** [comment]
I thought so too, when this first came up on the users' list, but couldn't find 
it.

**Comment by Adriani90** [comment]
Closing as duplicate of #7074.
